Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -64,20 +64,31 @@ public Task<ActionResult> GetCampaignThumbnail(string accountName, int campaignI
         [HttpPut("/accounts/{accountName}/campaigns/{campaignId}/content")]
         public async Task<IActionResult> SaveCampaign(string accountName, int campaignId, CampaignContent campaignContent)
         {
+            var htmlParser = new DopplerHtmlParser(campaignContent.htmlContent);
```

Comment:
Related to this, I prefer the other alternative present in [this PR] (https://github.com/FromDoppler/doppler-html-editor-api/pull/51)

Reply:
I do not 😛